lib/players/human.rb

```diff
@@ -0,0 +1,12 @@
+
+class Human < Player
+  attr_accessor :move
+  
+  def move(board)
+   input = gets
+   input
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(board)
  gets
end
```